user_manual/standard_unit.rst

```diff
@@ -0,0 +1,24 @@
+.. _standard-unit:
+
+Standard unit pecularities
+##########################
+
+In Ada, the ``Standard`` package is special: it is not a regular source file
```

Comment:
Also would be worth mentioning that ``Standard`` is at the root of every other package I guess.

Reply:
Doesn’t it have to be special, given that the character enums can’t be defined with regular Ada source code?

Reply:
If by special you mean "not a regular source file", I dunno, I can see other ways this could be expressed, something like: `type Character is new __Builtin_Character`, where naming conventions are relaxed by the compiler inside of `Standard`. That would allow users to customize the contents of `Standard`. Whether that's a good idea or not, I don't know :slightly_smiling_face: